gaphas/__init__.py

```diff
@@ -18,7 +17,25 @@
 used in Gaphas instead of the multiplier (``*``). In both the
 ``Canvas`` and ``View`` class a workaround is provided in case an
 older version of py-cairo is used.
+
+Copyright notice
+================
+
+Copyright 2018 Arjan Molenaar, Artur Wrobleski & Dan Yeaw
```

Comment:
Since @wrobell transferred his copyright to you, should we then remove him from the notice before we merge this?